projects/Amlogic/bootloader/install

```diff
@@ -50,7 +114,7 @@ case "${UBOOT_SYSTEM}" in
     cp $FIPDIR/lpddr4_1d.fw $DESTDIR/
     cp $FIPDIR/lpddr4_2d.fw $DESTDIR/
     cp $FIPDIR/piei.fw $DESTDIR/
-    cp u-boot.bin $DESTDIR/bl33.bin
+    cp $(get_build_dir $BOOTLOADER)/u-boot.bin $DESTDIR/bl33.bin
```

Comment:
Looks like you reintroduced that with a rebase, see 7d13537525faf278d0ca2c45e308e3af39b19c38